File: javascript-closure-angular/API/Client/RequestTimeFilterPostcodesArrivalSearch.js
goog.provide('API.Client.RequestTimeFilterPostcodesArrivalSearch');

/**
 * @record
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.id;

/**
 * @type {!API.Client.RequestTransportation}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.transportation;

/**
 * @type {!number}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.travelTime;

/**
 * @type {!Date}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.arrivalTime;

/**
 * @type {!Array<!API.Client.RequestTimeFilterPostcodesProperty>}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.properties;

/**
 * @type {!API.Client.RequestRangeFull}
 * @export
 */
API.Client.RequestTimeFilterPostcodesArrivalSearch.prototype.range;

File: scala-play-server/app/model/RequestTimeFilterPostcodesArrivalSearch.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime

/**
  * Represents the Swagger definition for RequestTimeFilterPostcodesArrivalSearch.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class RequestTimeFilterPostcodesArrivalSearch(
  id: String,
  transportation: RequestTransportation,
  travelTime: Int,
  arrivalTime: OffsetDateTime,
  properties: List[RequestTimeFilterPostcodesProperty],
  range: Option[RequestRangeFull]
)

object RequestTimeFilterPostcodesArrivalSearch {
  implicit lazy val requestTimeFilterPostcodesArrivalSearchJsonFormat: Format[RequestTimeFilterPostcodesArrivalSearch] = Json.format[RequestTimeFilterPostcodesArrivalSearch]
}

